SoftLayer/CLI/virt/create_options.py

```diff
@@ -6,213 +6,92 @@
 import SoftLayer
 from SoftLayer.CLI import environment
 from SoftLayer.CLI import formatting
-from SoftLayer import utils
 
 
 @click.command(short_help="Get options to use for creating virtual servers.")
+@click.option('--vsi-type', required=False, type=click.Choice(['TRANSIENT', 'SUSPEND']),
```

Comment:
Add `PUBLIC` as a choice, and make it the default here, instead of in the cli command. Add `show_default=True` to the option.